packages/prosemirror-lwdita-demo/src/demo-plugin.ts

```diff
@@ -152,9 +152,13 @@ function publishGithubDocument(config: Config, localization: Localization, urlPa
     if (dispatch) {
       dispatch(state.tr);
       const documentNode = transformToJditaDocumentNode(state);
+      // remove the base url from the documentNode
+      const baseUrl = urlParams.referer.split('/').slice(0, -1).join('/');
+      // doing the opposite of rawDoc.replace(/href="([^"]+)"/g, `href="${baseUrl}/$1"`);
```

Comment:
So we are removing the baseUrl from the html document? But I don't understand why?

Reply:
We are altering the xdita and adding the `baseURL` to it, when we serialize it at the end we need to remove our modification.